src/Models/Layout.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Plank\Contentable\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Plank\Contentable\Contracts\Layout as LayoutContract;
+
+class Layout extends Model implements LayoutContract
+{
+    use HasFactory;
+
+    protected $guarded = ['id'];
+
+    /**
+     * The key on the model which identifies the layout
+     */
+    const LAYOUT_KEY = 'identifier';
```

Comment:
is public not required anymore in PHP8.2?
```suggestion
    public const LAYOUT_KEY = 'identifier';
```

Reply:
or i suppose maybe its supposed to be scoped to this class only?

Reply:
TBH it can probably be defined inside the `public static function getLayoutKeyName()` method.